packages/babel-plugin-proposal-pipeline-operator/test/fixtures/pipeline-operator/async-with-transform-async/actual.js

```diff
@@ -0,0 +1,10 @@
+var incPromise = (x) => Promise.resolve(x + 1);
```

Comment:
i'd prefer to have the same code in actual / exec if they are in the same directory, exec differs slightly right now and it makes it hard to compare those files (assert ofc should stay just in exec)

Reply:
I made them match now. I think I may have saved the file with Prettier on at some point.

Reply:
Oh, I didnt mean that they do not match in sense of formatting - i think there was some differenc in how `incFuncPromise` was used

Reply:
Oh, got it. I also changed the usage of that variable down below. Either way, they're the same now.

Reply:
Cool!